### ISSUE TITLE
Nothing happens on clicking the icon

### ISSUE THREAD

**Issue by raoulteeuwen** [magnussuther/chrome-extension-saml-decoder#3]
Just installed the extension (thanks for developing it). When i click the icon, nothing happens. I can right-click it, and use the links to visit the extension-page in the Chrome store, but i don't get any window with SAML messages

**Comment by raoulteeuwen** [comment]
Follow up. De-installed and installed. I now do get to see a window when i click the icon. It says there are no saml messages to display, although i have refreshed a website where i login with saml twice... Shouldn't there be SAML-messages in the extension window?

**Comment by magnussuther** [comment]
Yeah, this is a known bug that everybody suffers from. I'v been consumed by work and haven't found the time to take a closer look. Shame on me. Also, I've been thinking of just rewriting the entire thing and drop dart. 

I've noticed that it is best to keep the list of SAML messages short, so hit the CLEAR ALL button often. That helps, and reduces the likelihood that this bug bites you. Sometimes, clicking the icon and waiting patiently for a minute or so brings up the list, after which you can click the CLEAR ALL button.
